cobc/pplex.l

```diff
@@ -1945,7 +1946,8 @@ start:
 	/* Skip comments that follow after AUTHORS, etc. */
 	if (unlikely (within_comment)) {
 		/* Check all of "Area A" */
-		for (ipchar = 7; ipchar < (n - 1) && ipchar < 11; ++ipchar) {
+		for (ipchar = cb_indicator_column;
+		     ipchar < (n - 1) && ipchar < cb_indicator_column + 4; ++ipchar) {
```

Comment:
please adjust to use something like the following before the `for` (it doesn't matter performance-wise here but is easier to follow)

```c
const int margin_b = cb_indicator_column + 4;
const int ipchar_max = (n - 1 < margin_b)  ? n - 1 : margin_b;
```